libs/boost/Makefile

```diff
@@ -368,10 +377,10 @@ define Host/Compile
 endef
 
 CONFIGURE_PREFIX:=$(PKG_INSTALL_DIR)
-TARGET_LDFLAGS += -pthread -lrt
+TARGET_LDFLAGS += -pthread -lrt -lstdc++ -Wl,--gc-sections,--as-needed,--print-gc-sections
```

Comment:
Do you have some insight as to why -lstdc++ is not added by the Make system?

Locally I'm having similar trouble when compiling pdns and friends with libcxx:

```
/home/mangix/devstuff/openwrt/staging_dir/toolchain-powerpc_8540_gcc-10.2.0_musl/lib/gcc/powerpc-openwrt-linux-musl/10.2.0/../../../../powerpc-openwrt-linux-musl/bin/ld: /home/mangix/devstuff/openwrt/staging_dir/target-powerpc_8540_musl/usr/lib/libboost_context.so: undefined reference to `__once_proxy'
/home/mangix/devstuff/openwrt/staging_dir/toolchain-powerpc_8540_gcc-10.2.0_musl/lib/gcc/powerpc-openwrt-linux-musl/10.2.0/../../../../powerpc-openwrt-linux-musl/bin/ld: /home/mangix/devstuff/openwrt/staging_dir/target-powerpc_8540_musl/usr/lib/libboost_context.so: undefined reference to `std::__once_callable'
/home/mangix/devstuff/openwrt/staging_dir/toolchain-powerpc_8540_gcc-10.2.0_musl/lib/gcc/powerpc-openwrt-linux-musl/10.2.0/../../../../powerpc-openwrt-linux-musl/bin/ld: /home/mangix/devstuff/openwrt/staging_dir/target-powerpc_8540_musl/usr/lib/libboost_context.so: undefined reference to `std::__throw_system_error(int)'
/home/mangix/devstuff/openwrt/staging_dir/toolchain-powerpc_8540_gcc-10.2.0_musl/lib/gcc/powerpc-openwrt-linux-musl/10.2.0/../../../../powerpc-openwrt-linux-musl/bin/ld: /home/mangix/devstuff/openwrt/staging_dir/target-powerpc_8540_musl/usr/lib/libboost_context.so: undefined reference to `std::__once_call'
collect2: error: ld returned 1 exit status
```

Reply:
Have no idea. It seems that it has something to do with the `b2` and the build system based on musl-libc. The boost libs on alpine have the same issue.

Reply:
Hmm OK.